src/components/alert/AccountNotInstalled.vue

```diff
@@ -67,7 +67,15 @@
          * @type {Event}
          */
         this.$emit('install', true);
+        this.$segment.track('ACC Click BO Install button', {
+          Category: 'Accounts',
+        });
       },
     },
+    mounted() {
+      this.$segment.track('ACC View Install component - install state', {
+        Category: 'Account',
```

Comment:
modulePrestaShop would be better . I know my documentation says category, but you can put whatever you want.